invenio_rdm_records/resources/serializers/dcat/__init__.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Datacite to DCAT serializer."""
+
+from datacite import schema43
+from lxml import etree as ET
+from pkg_resources import resource_stream
+from werkzeug.utils import cached_property
+
+from invenio_rdm_records.resources.serializers import DataCite43XMLSerializer
+
+
+class DCATSerializer(DataCite43XMLSerializer):
```

Comment:
This serializer is missing the serialized list (or does the inherited one works fine?), it should also be added in the tests [here](https://github.com/ppanero/invenio-rdm-records/blob/c88cbde27c432d732a4e49571725b27f959c25df/tests/resources/test_resources.py#L736) to make sure it works.